internal/util/pgbouncer.go

```diff
@@ -21,6 +21,11 @@ import (
 	crv1 "github.com/crunchydata/postgres-operator/pkg/apis/crunchydata.com/v1"
 )
 
+// pgBouncerConfigMapFormat is the format used for the name of the config
+// map associated with a pgBouncer cluster, and follows the pattern
+// "<clusterName>-pgbouncer-cm"
+const pgBouncerConfigMapFormat = "%s-pgbouncer-cm"
```

Comment:
Wondering if we should change the name to `<clusterName>-pgbouncer-config` to align with our other ConfigMap naming, e.g.:

- `<clusterName>-config` for the Patroni DCS
- `<clusterName>-pgha-config` for the customizing Patroni and PG configs
- `pgo-config` for Operator configuration

Reply:
@andrewlecuyer I had thought about that, but I also remembered the thread about the names being too long, so I wanted to shorten it slightly.